### DISCUSSION TITLE
Backslash in block quotes causes errors

### DISCUSSION THREAD

**Discussion by csrmy** [Witiko/markdown#494] [Q&A]
Hi,
I am new to Tex and LaTex.

@Witiko let me know if this should be in [issues](https://github.com/Witiko/markdown/issues) instead of here.
When using backslash for new line in a block quote, it causes long series of errors.

Test tex file:
```
\documentclass{article}
\usepackage{markdown}

\begin{document}
\begin{markdown}
> **TITLE** \
> Lorem ipsum dolor sit amet
\end{markdown}
\end{document}
```

Thank you in advance for your time and help.

**Comment by Witiko** [comment]
Closed via https://github.com/Witiko/markdown/pull/496.
